ratelimit/redis.go

```diff
@@ -52,6 +54,7 @@ func newClusterRateLimiterRedis(s Settings, r *net.RedisRingClient, group string
 		window:     s.TimeWindow,
 		ringClient: r,
 		metrics:    metrics.Default,
+		sometimes:  rate.Sometimes{First: 3, Interval: 50 * time.Millisecond},
```

Comment:
What is the reasoning behind 50ms? Why not 1s?

Reply:
This interval defines how frequently we add span with errors. I though to use 50 ms instead of second because the error may be occuring for a short time and we can miss it if we set to once a sec. I am flexible with changing it to few hundred ms as well.

Reply:
I do not think we should care too much for short error spikes - these would be visisble as error spans. For prolonged (several seconds) failures we would see error logs and we do not want to flood logs. I would say 1s is a good round number to set as a rate limit.